src/views/app/detail-panel/edit/edit-view.jsx

```diff
@@ -344,20 +344,23 @@ export default function EditView({ mailId, folderId, setHeader, toggleAppBoard }
 		}
 	}, [action, boardContext, editor, t, dispatch, editorId, folderId, replaceHistory, closeBoard]);
 
-	const throttledSaveToDraft = (data) => {
-		clearTimeout(timer);
-		const newTimer = setTimeout(() => {
-			const newData = { ...editor, ...data };
-			if (saveFirstDraft) {
-				saveDraftCb(newData);
-				setSaveFirstDraft(false);
-			} else if (editor.id && editor.id !== 'undefined') {
-				saveDraftCb(newData);
-			}
-		}, 500);
+	const throttledSaveToDraft = useCallback(
+		(data) => {
+			clearTimeout(timer);
+			const newTimer = setTimeout(() => {
+				const newData = { ...editor, ...data };
+				if (saveFirstDraft) {
```

Comment:
Yes,It's useful because when you do reply or forward to a message the editor will initialise and the input values will change and it will immediately trigger the save draft request.So, I'm doing this to skip the auto save for initialising of editor